vpr/src/draw/intra_logic_block.cpp

```diff
@@ -34,11 +34,38 @@
 #include "draw.h"
 #include "draw_triangle.h"
 
+constexpr float FRACTION_TEXT_PADDING = 0.01;
+
 /************************* Subroutines local to this file. *******************************/
 
+/**
+ * @brief A recursive function which traverses through each sub-block of a pb_graph_node, calculates bounding boxes for each sub-block, and stores the bounding boxes in the global draw_coords variable.
+ * @param type_descrip_index The index of the logical block type.
+ * @param pb_graph_node The pb_graph_node.
+ * @param parent_width The width of the parent block.
+ * @param parent_height The height of the parent block.
+ */
 static void draw_internal_load_coords(int type_descrip_index, t_pb_graph_node* pb_graph_node, float parent_width, float parent_height);
+
+/**
+ * @brief Finds the maximum depth of sub-blocks for a given pb_type.
+ * @param pb_type The top-level pb_type.
+ * @return The maximum level for the given pb_type.
+ */
 static int draw_internal_find_max_lvl(const t_pb_type& pb_type);
-static void draw_internal_calc_coords(int type_descrip_index, t_pb_graph_node* pb_graph_node, int num_pb_types, int type_index, int num_pb, int pb_index, float parent_width, float parent_height, float* blk_width, float* blk_height);
+/**
+ * @brief A helper function for draw_internal_load_coords. Calculates the coordinates of a internal block and stores its bounding box inside global variables. The calculated width and height of the block are also assigned to the pointers blk_width and blk_height.
```

Comment:
Line is too long; you should break it into multiple lines so it displays nicely.